### ISSUE TITLE
Needs update for patch 1.18.13.0

### ISSUE THREAD

**Issue by fleps** [mixMugz/msfs2020-uimod#9]
Hello

I didn't tested every single details, but just letting you know that while the mods do not crash the game, they don't fully work.

- [uimod]-mugz-tinyplates seems to work fine, plates are smaller and you can hide them with F12
- nunlock to hide other things do not work
- backspace hides only POI names now
- POI are smaller with [uimod]-mugz-tinypoi / [uimod]-mugz-tinyhud, but the lines / icons do not hide anymore
- external HUD also do not hide at all

Hope this can be fixed soon, your mods are a must have!

Thanks

**Comment by mixMugz** [comment]
Yes it's planed, but i'm not sure what all addons will work due asobo limitation
- UI mod, need some changes
- Need to rework
- Need to rework
- POI you can remove, default now look good
- Also need to rework

**Comment by DrFeelgood66** [comment]
Can we expect and update soon or is this mod dead? 

It has not been updated since last year :(

**Comment by mixMugz** [comment]
Dont have a lot time, but planed to update

**Comment by sonicviz** [comment]
Do you know if it's just possible to get the hide labels etc working again? That's really useful for screenshots.

**Comment by mixMugz** [comment]
> Do you know if it's just possible to get the hide labels etc working again? That's really useful for screenshots.

I didnt find any way to allow work again hide tooltip

**Comment by mixMugz** [comment]
Updated to 1.21.18.0, but not full collection
